packages/core/src/util/requireHook.js

```diff
@@ -55,9 +55,12 @@ exports.init = function (config) {
  */
 function patchedModuleLoad(moduleName) {
   // CASE: when using ESM, the Node runtime passes a full path to Module._load
-  //       we try to grab the module name to being able to patch the target module
-  //       with our instrumentation
+  //       We aim to extract the module name to apply our instrumentation.
   // CASE: we ignore all file endings, which we are not interested in. Any module can load any file.
+  // CASE: The native ESM support is pending, and the requireHook isn't compatible with native ESM.
+  //       However, 'got' is automatically instrumented along with the the http2wrapper,
```

Comment:
```suggestion
  //       However, 'got' is instrumented by coincidency, because it is requiring the [http2wrapper](https://github.com/search?q=repo%3Asindresorhus%2Fgot%20from%20%27http2-wrapper%27&type=code) CJS package, which is requiring http.
```
```